config/routes.rb

```diff
@@ -1,18 +1,17 @@
 Rails.application.routes.draw do
   devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  devise_scope :user do
-    authenticated :user do
-      root :to => "users#index", as: :authenticated_root
-    end
-    unauthenticated :user do
-      root :to => "devise/sessions#new", as: :unauthenticated_root
-    end
-  end
+  root "users#index"
+  # Define endpoints for
+  post "/api/register", to: 'authentication#register'
+  post '/api/login', to: 'authentication#login'
+  get '/api/users/:user_id/posts', to: 'posts#posts'
+  get '/api/posts/:id/comments', to: 'comments#comments'
+  post '/api/comments', to: 'comments#add_comment'
```

Comment:
- [ ] You should also have an API endpoint that returns all comments for a post - GET : `/users/:user_id/posts/:id/comments`
- [ ] Let's add an API endpoint that adds' a comment to a post - POST : `/users/:user_id/posts/:id/comments`